web/auth.go

```
@@ -0,0 +1,49 @@
+	}
+}
+
+func AuthLogin(c *gin.Context, cfg *config.Config) {
+	username := c.PostForm("username")
+	password := c.PostForm("password")
+
+	// 这里应该验证用户名和密码
+	if username == cfg.Auth.Username && password == cfg.Auth.Password {
+		session := sessions.Default(c)
+		session.Set("authenticated", true)
+		session.Save()
```

golangci-lint (1.62.2)

Error return value of `session.Save` is not checked

(errcheck)